File: src/main/java/net/server/MigrateInfo.java
package net.server;

import lombok.Getter;
import lombok.NonNull;
import lombok.RequiredArgsConstructor;
import lombok.Setter;

@RequiredArgsConstructor
@Getter
@Setter
public class MigrateInfo {

    private @NonNull int aid, channel;
    private @NonNull String ip;
    private int cid;
    private boolean cashShop;

    @Override
    public String toString() {
        return "MigrateInfo{" +
                "aid=" + aid +
                ", channel=" + channel +
                ", ip='" + ip + '\'' +
                ", cid=" + cid +
                ", cashShop=" + cashShop +
                '}';
    }
}

File: src/main/java/net/server/MigrateInfo.kt
package net.server

class MigrateInfo(
    val aid: Int,
    var port: Int,
    val ip: String
) {

    var channelId: Int = -1
    var cashShop = false

    override fun toString(): String {
        return "MigrateInfo(aid=$aid, channel=$channelId, ip='$ip', cashShop=$cashShop)"
    }
}